frontend/src/CodeBlock/atelier-forest-light.css

```
@@ -0,0 +1,69 @@
+/* Base16 Atelier Forest Light - Theme */
+/* by Bram de Haan (http://atelierbram.github.io/syntax-highlighting/atelier-schemes/forest) */
+/* Original Base16 color scheme by Chris Kempson (https://github.com/chriskempson/base16) */
```

Let's change the header so it is clear what copyrights are in place and what under this is licensed. As I saw the original works are MIT licensed and this is compatible with our BSD license, it is okay to re-license under BSD. This way we keep the variaty of licenses in the project lower.

For dependencies I usually would not recommend this, but in this case we "import" (copy) the source code into ours and (assuming this took place) modify it.

https://github.com/chriskempson/base16/blob/master/LICENSE.md
https://github.com/atelierbram/syntax-highlighting

```suggestion
/*!
 * Copyright 2020 Marius Wilms. All rights reserved.
 * Copyright 2019 Atelier Disko. All rights reserved.
 * Coypright XXXX Bram de Haan. All rights reserved.
 * Copyright XXXX Chris Kempson. All rights reserved.
 *
 * Use of this source code is governed by a BSD-style
 * license that can be found in the LICENSE file.
 */
```